Form-Controls/index.html

```diff
@@ -9,19 +9,67 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
+    <h2 class="para">Please fill in the order form to complete your order</h2>
+
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <div class="form">
+    <label for="fname">First Name:</label><br>
+    <input type="text" id="fname" name="fname" pattern="[A-Za-z]+" title="Only letters are allowed" required/>
+    <br>
+    <label for="lname">Last Name:</label><br>
+    <input type="text" id="lname" name="lname" pattern="[A-Za-z]+" title="Only letters are allowed" required/>
+    <br>
+    <label for="email">Email:</label><br>
+    <input id="email" type="email" required/>
+  </div>
+
+  <div class="shirt-colours">
+    <fieldset>
+      <legend>Choose Colors:</legend>
+    
+      <input id="white" type="radio" name="colour" value="white"/>
+      <label for="white">White</label>
+    
+      <input id="red" type="radio" name="colour" value="red"/>
+      <label for="red">Red</label>
+    
+      <input id="black" type="radio" name="colour" value="black"/>
+      <label for="black">Black</label>
+    </fieldset>
+  </div>
+
+  <div class="shirt-size">
```

Comment:
I notice that I can submit the form without choosing a size option. How should we handle this case?

Reply:
fixed